plugins/ruby-foreman-templates/debian/changelog

```diff
@@ -1,3 +1,9 @@
+ruby-foreman-templates (9.3.0-3) stable; urgency=low
+
+  * 9.3.0-3 released
```

Comment:
You can change this to something like "update bundled dependencies for security fixes"